docker-compose.yml

```diff
@@ -40,6 +40,7 @@ services:
       - MONGODB_URI=mongodb://root:r00t@dtfs-submissions-data:27017/?authMechanism=DEFAULT
       - DTFS_CENTRAL_API=http://localhost:5005
       - TZ=Europe/London
+      - NODE_ENV=development
```

Comment:
Can this and all of the below be picked up from env please `${{ env.NODE_END }}`

Reply:
good shout - this is updated now